core/commonMain/src/kotlinx/io/bytes/BytesOperations.kt

```diff
@@ -0,0 +1,25 @@
+package kotlinx.io.bytes
+
+import kotlinx.io.*
+import kotlinx.io.buffer.*
+import kotlin.contracts.*
+
+/**
+ * Create [BytesInput] with content from [block] using specified [bufferSize].
+ */
+fun buildInput(bufferSize: Int = DEFAULT_BUFFER_SIZE, block: BytesOutput.() -> Unit): BytesInput {
```

Comment:
explicit public

Reply:
And the same comment about usefulness of this primitive that is not reflected in the documentation